build.gradle

```diff
@@ -3,16 +3,16 @@ plugins {
     id 'checkstyle'
     id 'pmd'
     id 'jacoco'
-    id 'com.github.johnrengelman.shadow' version '6.1.0'
-    id 'nebula.lint' version '17.7.0'
-    id "org.owasp.dependencycheck" version "7.2.1"
+    id 'maven-publish'
```

Comment:
why maven publish? do we need to deploy it to a repository somewhere? do we use it as a library?